sample/core/src/main/kotlin/br/com/zup/beagle/sample/GlobalContext.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.sample
+
+import br.com.zup.beagle.annotation.GlobalContext
+import br.com.zup.beagle.sample.builder.Person
+
+@GlobalContext
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/1767037c5777b4b44c2b78a548a8a8a8514b5d6b/sample/core/src/main/kotlin/br/com/zup/beagle/sample/GlobalContext.kt#L22"></span>The file name 'GlobalContext' does not match the name of the single top-level declaration 'GlobalObject'.</td>
    </tr>
  </tbody>
</table>